packages/docusaurus-utils/src/index.ts

```diff
@@ -173,19 +173,19 @@ export function normalizeUrl(rawUrls: string[]): string {
     urls[0] = first + urls[0];
   }
 
-  // There must be two or three slashes in the file protocol, two slashes in anything else.
-  if (urls[0].match(/^file:\/\/\//)) {
-    urls[0] = urls[0].replace(/^([^/:]+):\/*/, '$1:///');
-  } else {
-    urls[0] = urls[0].replace(/^([^/:]+):\/*/, '$1://');
-  }
+  /**
```

Comment:
```
/**
 *
 */
```

The syntax is reserver for file or function level comments. Inline comments just use `//`.